scripts/hud/cgaz.ts

```diff
@@ -737,15 +841,15 @@ class Cgaz {
 			}
 			this.pitchStat.style.color = bShouldHighlight ? this.compassHlColor : this.compassColor;
 		}
-		this.pitchStat.visible = this.compassStatMode % 2;
+		this.pitchStat.visible = this.compassStatMode % 2 !== 0;
```

Comment:
just for my own interest, is this better than `Boolean(this.compassStatMode % 2)`?

Reply:
Performance-wise I don't know, though I suspect the direct comparison is faster. I find the code much clearer doing the comparison, since modulo checks like this are so common in every language, and doesn't require reasoning about truthiness of different numbers.